src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
```

Comment:
Please 
```
@echo off 
setlocal 
```

and 

```
endlocal
``` 

at the end 

If you're going to show status messages, please explicitly `echo` them rather than relying on REM to show the comments.